### PR TITLE
Usability improvements

### PR DESCRIPTION
**Pull request by S-Dafarra**
Waiting for #119 

This code has been used for teleoperation experiments where just the neck needed to be retargeted

### CODE REVIEW

**Comment by S-Dafarra** [comment]
This PR is ready to be reviewed

**Comment by S-Dafarra** [comment]
Friendly ping @GiulioRomualdi

File: modules/Oculus_module/src/OculusModule.cpp [hunk 1]
```diff
@@ -956,8 +1085,8 @@ bool OculusModule::updateModule()
         // get the transformation form the oculus
         if (!getTransforms())
         {
-            yError() << "[OculusModule::updateModule] Unable to get the transform";
-            return false;
+            yError() << "[OculusModule::updateModule] Unable to get the transforms. Skipping iteration.";
```
Comment (GiulioRomualdi) [inline modules/Oculus_module/src/OculusModule.cpp:1088]: should we change it to a warning?

**Review by GiulioRomualdi — COMMENTED** [review]
Just a minor comment

File: modules/Oculus_module/src/OculusModule.cpp (hunk 1 above)
Comment (S-Dafarra) [reply modules/Oculus_module/src/OculusModule.cpp]: Done in https://github.com/robotology/walking-teleoperation/pull/121/commits/ea931bad41b2795a31f7a419d2dbfc74b10909d3

**Comment by GiulioRomualdi** [comment]
Feel free to merge it 😄

**Comment by S-Dafarra** [comment]
Thanks @GiulioRomualdi, merging

**Pull request merged**